apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -90,9 +92,11 @@ const providers: Provider[] = [
         username: user.username,
         email: user.email,
         name: user.name,
+        role: user.role,
```

Comment:
Would it also work with other auth providers like Google, SAML? I am not sure so just asking this question for confirmation.

Reply:
It should - based off they all use the same JWT callback in next-auth. Ill setup them both locally and get back to you

Reply:
Yes @hariombalhara I think it should work with those too, since the role level is at the database/model level. As long as we expose the role model to next-auth, all sessions should get this. We need to test it obviously.